### DISCUSSION TITLE
Cannot change page slug name

### DISCUSSION THREAD

**Discussion by VoxNumana** [GetPublii/Publii#1704] [Help] [answered]
Hi, had an 'About' post which I had been previously using as a 'page'. Now that Publii can use pages, I deleted my 'About' post and created an 'About' page. However, the page slug shows 'about-2' and no amount of updating the slug will change it.

I deleted the page and created a new one, which now shows the slug as 'about-3'. I cannot rename this slug either. Anybody know how I can change the slug back to 'about'?

**Comment by bobmitro** [comment, marked as answer]
It seems that you just moved the post to the trash, so the post still exists, so you cannot create another post with the same slug. To fix this, permanently delete the post from the trash.

**Reply by VoxNumana** [reply]
That make sense. It works. Thanks for your prompt answer.
